apis/nucleus/src/components/Cell.jsx

```diff
@@ -284,6 +285,8 @@ const Cell = forwardRef(({ halo, model, initialSnOptions, initialError, onMount
   const [selections] = useObjectSelections(app, model);
   const [hovering, setHover] = useState(false);
   const hoveringDebouncer = useRef({ enter: null, leave: null });
+  const [dimensionList] = useLibraryList(app, 'dimension');
```

Comment:
Including these hooks in here - does that mean the dimension and measure list will get fetched every time a chart is initialized?